packages/vercel/vercel.0.1.0/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+maintainer: "Antonio Nuno Monteiro <anmonteiro@gmail.com>"
+authors: [ "Antonio Nuno Monteiro <anmonteiro@gmail.com>" ]
+license: "BSD-3-clause"
+homepage: "https://github.com/anmonteiro/aws-lambda-ocaml-runtime"
+bug-reports: "https://github.com/anmonteiro/aws-lambda-ocaml-runtime/issues"
+dev-repo: "git+https://github.com/anmonteiro/aws-lambda-ocaml-runtime.git"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" {>= "4.08"}
+  "dune" {>= "1.7"}
+  "result"
+  "yojson"
+  "lwt"
+  "base64"
+  "piaf"
+  "ppx_deriving_yojson"
+  "lambda-runtime"
```

Comment:
```suggestion
  "lambda-runtime" {=version}
```

we tend to enforce strict bounds on packages that come together from the same repository. Before we did this, bounds were getting messy and metadata was getting hard to manage. Would this be a problem for you? Do you prefer keeping this unconstrained for some specific reason?

Reply:
This should be OK. I'll fix it going forward in the upstream repo, if that works.